src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Hello</title>
+  </head>
```

Comment:
Will be great to add other required tags

Reply:
Got it, I've put some staff there!